07-routing-gossip.md

```diff
@@ -647,6 +692,21 @@ timeouts.  It also causes a natural ratelimiting of queries.
     * [`chain_hash`:`chain_hash`]
     * [`u32`:`first_blocknum`]
     * [`u32`:`number_of_blocks`]
+    * [`byte`:`option_extended_query_flag`]
+    * [`byte`:`query_flag`] (`option_extended_query_flags`)
+    * [`query_channel_range_tlv`]
+      * type: 1 (`query_option`)
+      * data:
+        * [`byte`:`query_option_flags`]
```

Comment:
this definition says that the type is `byte`, but elsewhere it is described as a minimally encoded varint. which varint encoding is being proposed?

Reply:
Right, it should be a  `varint`  (using the LN definition of `varint`)